src/Entity.php

```diff
@@ -2166,6 +2177,20 @@ public static function showNotificationOptions(Entity $entity)
         }
 
         echo "</td></tr>";
+        echo "<td>" . __('Reminders frequency for alarms on low printer cartridge levels') . "</td><td>";
+        $default_value = $entity->fields['printer_cartridge_levels_alert_repeat'];
+        Alert::dropdown(['name'           => 'printer_cartridge_levels_alert_repeat',
+            'value'          => $default_value,
+            'inherit_parent' => (($ID > 0) ? 1 : 0)
+        ]);
+
+        if ($entity->fields['printer_cartridge_levels_alert_repeat'] == self::CONFIG_PARENT) {
+            $tid = self::getUsedConfig('printer_cartridge_levels_alert_repeat', $entity->getField('entities_id'));
+            self::inheritedValue(self::getSpecificValueToDisplay('printer_cartridge_levels_alert_repeat', $tid), true);
+        }
+
+        echo "</td></tr>";
+
```

Comment:
originally, the "Cartridge" option in Entity > notification mainly concerned stock management, 
adding an option here for cartridge level alerts bothers me.

![image](https://github.com/glpi-project/glpi/assets/7335054/2f74c775-9759-4c27-8c4a-fc44f23f1df2)


For me, it's necessary to add a new "line" "cartridge level alerts" and possibly rename the first "cartridge" line as "Cartridge stock".